app/src/main/java/io/multy/Multy.java

```diff
@@ -42,6 +43,12 @@ public void onCreate() {
 
         context = getApplicationContext();
 
+        Realm.setDefaultConfiguration(new RealmConfiguration.Builder()
```

Comment:
Setting realm configuration must be set only after generating separate key for DB. Please read code with attention.